lib/htmlValidator.js

```diff
@@ -186,11 +198,35 @@ module.exports = function (app, callback) {
         return validationMessage
       }
 
-      if (req.headers[headerException]) {
-        res.set(headerException, true)
+      function validatorExceptions () {
+        let found = false
+
+        if (foundModelException) {
+          found = true
```

Comment:
why bother with the rest of the function? just return true here

Reply:
Good point. Get rid of the found variable and just replace every instance of `found = true` with `return true` and then replace drop a `return false` under the exception checks.